Dockerfiles/Python/Dockerfile

```diff
@@ -51,3 +57,4 @@ RUN pip3 install -r requirements-app.txt
 RUN pip3 install -r requirements-tests.txt
 # Keep root as last user
 USER root
+WORKDIR /
```

Comment:
is this needed? it was also working before or?

Reply:
Was working before when you enter the image interactively, you end up in /requirements which is not too great